test/integration/plugin/test_projectpath.py

```diff
@@ -19,7 +19,10 @@ def test_project_path_from_toml_config(pytester):
         def test_arch(arch_project_paths, pytestconfig):
             assert len(arch_project_paths) == 2
             assert arch_project_paths[0] == pytestconfig.rootpath / 'foobar'
-            assert arch_project_paths[1] == Path('/foo/bar')
+            drive = ''
+            if arch_project_paths[1].drive:
+                drive = arch_project_paths[1].drive
```

Comment:
I think you can replace this with just `drive = arch_project_paths[1].drive`?
On non-Windows systems drive should be an empty string.